cmd/yurthub/app/config/config.go

```diff
@@ -308,6 +308,10 @@ func registerInformers(options *options.YurtHubOptions,
 		informerFactory.InformerFor(&corev1.Secret{}, newSecretInformer)
 	}
 
+	newPodInformer := func(client kubernetes.Interface, resyncPeriod time.Duration) cache.SharedIndexInformer {
+		return coreinformers.NewPodInformer(client, "", resyncPeriod, nil)
+	}
+	informerFactory.InformerFor(&corev1.Pod{}, newPodInformer)
```

Comment:
we need new pod informer when working mode is cloud, so we can add a condition as following in order to reduce costs for yurthub.

```
if workingMode == util.WorkingModeCloud {
	newPodInformer := func(client kubernetes.Interface, resyncPeriod time.Duration) cache.SharedIndexInformer {
		return coreinformers.NewPodInformer(client, "", resyncPeriod, nil)
	}
	informerFactory.InformerFor(&corev1.Pod{}, newPodInformer)
}
```